load/dependencyresolver.go

```diff
@@ -33,9 +35,9 @@ type dependencyGenerator func(node *parse.Node) ([]string, error)
 // the graph and creates edges to fit them
 func ResolveDependencies(ctx context.Context, g *graph.Graph) (*graph.Graph, error) {
 	logger := logging.GetLogger(ctx).WithField("function", "ResolveDependencies")
-	logger.Debug("resolving dependencies")
+	logger.Info("resolving dependencies")
 
-	return g.Transform(ctx, func(id string, out *graph.Graph) error {
+	r, e := g.Transform(ctx, func(id string, out *graph.Graph) error {
```

Comment:
I know it's been brought up before that single-letter variable names can make things hard to read. not sure it's a problem here since you are just returning them (don't even really need them).